py3status/modules/bluetooth.py

```diff
@@ -55,47 +52,40 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     device_separator = '|'
-    format = '{name}'
-    format_no_conn = 'OFF'
-    format_no_conn_prefix = 'BT: '
-    format_prefix = 'BT: '
+    format = 'BT[: {format_device}]'
+    format_device = '{name}'
+
+    class Meta:
+        def deprecate_function(config):
+            out = {}
+            if 'format_prefix' in config:
+                out['format'] = u'{}{{format_device}}'.format(config['format_prefix'])
+            return out
```

Comment:
Damn our threading model. There must be something we can use... even nasty hacks. It's better than nothing. Maybe we can use an average number out of 100 one-shots or something (excluding API modules). I don't know.

Anyway, here to say we should decide soon what to do with this. Do we mark this ready or we come up with more than just a prefix fix since this new format is much more cleaner than the old format.